src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx

```diff
@@ -137,11 +120,37 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
     this.props.navigator.pop()
   }
 
+  getCameraRollPhotos = (params: any) => CameraRoll.getPhotos(params)
```

Comment:
RN 0.48 did not include the Jest mapping for the camera, so by moving that call into this easily replaced function, it becomes easy to test